### ISSUE TITLE
Support BitSet

### ISSUE THREAD

**Issue by GrapeBaBa** [gballet/ssz.zig#13]
Can we support BitSet type? It is mostly used by consensus spec.

**Comment by gballet** [comment]
sorry I didn't see your message. Doesn't `[]bool` do what you want ?

**Comment by GrapeBaBa** [comment]
Yes, we use it right now, but we used zig master version and forked this repo

**Comment by gballet** [comment]
Fixed in 0.0.3, closing.
